com.haysell.pos.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id":"com.haysell.pos",
```

Comment:
https://haysell.com/ says page not found. How are we supposed to verify that you own the domain?

Reply:
Our service is currently available from Armenia ( and possible from somewhere nearly )
This is maybe some 'cloudflaer' setting, which I can not change ( I'm developing Point Of Sale application only )
from my machine haysell.com is available.
is this issue critical ? 
![Screenshot from 2024-07-27 15-41-51](https://github.com/user-attachments/assets/78f8312f-c73d-454e-9856-010c5389e7e0)

Reply:
>  is this issue critical ?

Yes

I can't access https://en.haysell.com/ either.

Reply:
works fine here, fwiw. Maybe it's a problem with your geographical location.

Reply:
or just clownflare ddos "protection"

Reply:
Yea seems to be cloudflare blocking, it worked after using a VPN.

@Delphi251189 I don't see any mention of https://github.com/Delphi251189/haysell-pos.git on the webpage. Can you link it somewhere on the website?

Reply:
Basically, yes, this limitation is due to cloudflare DDOS protection, and I can't change anything in backoffice, because it's developing with other team, and fully separated from PoS.
It is easier to think that there is no backoffice at all.
Let's say haysell-pos is a stand-alone application and have no web sites/domains/addresses and anything web-related ( just like media-player).
 What about such scenario ?  Do I need to confirm ownership of a domain if I don't have it at all?

Reply:
>  Do I need to confirm ownership of a domain if I don't have it at all?

You need to always confirm ownership of the domain, otherwise anyone can use a random domain and say it is theirs.

Reply:
ok, in such case dl.haysell.com subdomain is controlled by me ( actually, just downloads location, where I push app binaries ).

I can create a **basic and simple html** (I'm not so good at HTML/CSS/JS) where I can put link to github repo.
this is good enough

Reply:
Just for confirmation

https://dl.haysell.com/downloads/pos/sources.html

Reply:
> actually, just downloads location, where I push app
please note, that it is not full-functional web page or something similar.
Instead, it just running Nginx Web Server in HTTP Server mode...in other words, simplly serving files in HTTP and SFTP modes, noting more.
accordingly, the page with github repo link is not the most luxurious